src/vs/workbench/contrib/terminal/browser/links/terminalLinkManager.ts

```diff
@@ -229,6 +231,13 @@ export class TerminalLinkManager extends DisposableStore {
 			}
 		}
 
+		let fallbackLabel: string;
+		if (this._tunnelService.canTunnel(URI.parse(uri))) {
+			fallbackLabel = nls.localize('followForwardedLink', "Follow Link for Forwarded Port");
```

Comment:
Suggestion:

> Follow Forwarded Port Link

or

> Follow Link using Forwarded Port

Reply:
Going with "Follow Link using Forwarded Port" since I find it more descriptive.